### PR TITLE
Update open api json

### PR DESCRIPTION
**Pull request by zsluedem**
## Overview

We instroduce new api in https://github.com/rchain/rchain/releases/tag/v0.12.2 while the api is not updated in https://developer.rchain.coop/rnode-api.

Update the json to update the api in developer website

You can easily go to https://petstore.swagger.io/ and past the json url of this commit and try.
### Please make sure that this PR:
- [x] is at most 200 lines of code (excluding tests),
- [x] meets [RChain development coding standards](https://rchain.atlassian.net/wiki/spaces/DOC/pages/28082177/Coding+Standards),
- [x] includes tests for all added features,
- [x] has a reviewer assigned,
- [x] has [all commits signed](https://rchain.atlassian.net/wiki/spaces/DOC/pages/498630673/How+to+sign+commits+to+rchain+rchain).

### [Bors](https://bors.tech/) cheat-sheet:

- `bors r+` runs integration tests and merges the PR (if it's approved),
- `bors try` runs integration tests for the PR,
- `bors delegate+` enables non-maintainer PR authors to run the above.

### CODE REVIEW

File: docs/rnode-api/rnode-openapi.json [hunk 1]
```diff
@@ -38,9 +38,13 @@
           "description": "WebApi.scala",
           "url": "https://github.com/rchain/rchain/blob/dev/node/src/main/scala/coop/rchain/node/api/WebApi.scala"
         },
-        "tags": ["blocks"],
+        "tags": [
```
Comment (stanislavlyalin) [inline docs/rnode-api/rnode-openapi.json]: Are these changes correspond to API presented in the code? Are they generated from code or should be written manually?

Comment (zsluedem) [reply docs/rnode-api/rnode-openapi.json]: > Are these changes correspond to API presented in the code?

Yes. We created a new api endpoint `/api/transaction/{hash}` at https://github.com/rchain/rchain/pull/3433 which is not covered in current openapi json and that's why we need this pr.

> Are they generated from code or should be written manually?

Currently no. But they should be auto-matically generated when we implement a new api.  I used to have a pr https://github.com/rchain/rchain/pull/3128 which is about generating these open API json. But the auto-generated codes are not accurate so @dckc manually code all these things and made a [pr](https://github.com/rchain/rchain/pull/3140) for that.

**Our goal in the future is to fix this and make these auto-generated with handwriting.**

Comment (dckc) [reply docs/rnode-api/rnode-openapi.json]: Please open an issue to track the automation goal.

File: docs/rnode-api/rnode-openapi.json [hunk 2]
```diff
@@ -38,9 +38,13 @@
...
+          "blocks"
+        ],
         "operationId": "getBlocks",
-        "produces": ["application/json"],
+        "produces": [
+          "application/json"
+        ],
```
Comment (dckc) [inline docs/rnode-api/rnode-openapi.json]: Why the whitespace changes? The previous style was from a vs-code plug-in (I suppose documenting that only in a commit message wasn't very thorough).

If you really want to keep them, could you put them in their own commit?

File: docs/rnode-api/rnode-openapi.json (hunk 1 above)
Comment (dckc) [reply docs/rnode-api/rnode-openapi.json]: > We created a new api endpoint /api/transaction/{hash} at #3433 which is not covered in current openapi json

Please make that more clear in the commit message; for example:

```
fix: /api/transaction/{hash} was missing from open api definition
```

**Review by dckc — CHANGES_REQUESTED** [review]
Please either remove the whitespace changes or document the rationale.

File: docs/rnode-api/rnode-openapi.json (hunk 1 above)
Comment (zsluedem) [reply docs/rnode-api/rnode-openapi.json]: > Please open an issue to track the automation goal.

https://github.com/rchain/rchain/issues/2974

File: docs/rnode-api/rnode-openapi.json (hunk 2 above)
Comment (zsluedem) [reply docs/rnode-api/rnode-openapi.json]: Sorry for the confusion

Comment (zsluedem) [reply docs/rnode-api/rnode-openapi.json]: Actually what was the formatter you used?

**Review by dckc — APPROVED** [review]

**Comment by zsluedem** [comment]
@dckc Thanks very much for the review. I always appreciate your wisdom. 🥇

**Review by tgrospic — APPROVED** [review]
LGTM

File: docs/rnode-api/rnode-openapi.json (hunk 2 above)
Comment (dckc) [reply docs/rnode-api/rnode-openapi.json]: The formatter was part of https://github.com/42Crunch/vscode-openapi

**Comment by zsluedem** [comment]
bors merge

**Review by nzpr — APPROVED** [review]
LGTM

**Comment by nzpr** [comment]
bors merge
